examples/hybrid/hyperdiffusion.jl

```diff
@@ -0,0 +1,78 @@
+hyperdiffusion_cache(
+    ᶜlocal_geometry,
+    ᶠlocal_geometry;
+    κ₄ = FT(0),
+    use_tempest_mode = false,
+) = merge(
+    (;
+        ᶜχ = similar(ᶜlocal_geometry, FT),
+        ᶜχuₕ = similar(ᶜlocal_geometry, Geometry.Covariant12Vector{FT}),
+        κ₄,
+        use_tempest_mode,
+    ),
+    use_tempest_mode ? (; ᶠχw_data = similar(ᶠlocal_geometry, FT)) : (;),
+)
+
```

Comment:
The unicode is over-used here. `div`, `grad`, `curl`, and their weak forms by adding a `w` in front is explanatory enough and very easy to read. I think we should be careful and not abuse unicode